pkg/envoy/grpc.go

```diff
@@ -289,29 +297,46 @@ func (s *RDSServer) translatePolicyRule(h api.PortRuleHTTP) *envoy_api.Route {
 		cnt++
 	}
 
+	var rule_ref string
 	isRegex := wrappers.BoolValue{Value: true}
 	headers := make([]*envoy_api.HeaderMatcher, 0, cnt)
 	if h.Path != "" {
 		headers = append(headers, &envoy_api.HeaderMatcher{Name: ":path", Value: h.Path, Regex: &isRegex})
+		rule_ref = "PathRegexp(\"" + h.Path + "\")"
```

Comment:
String literal tip I gave back can be applied here.

Reply:
Yes, thanks!